src/ui/pages/ConnectionDetails/components/EditConnectionsModal.tsx

```diff
@@ -48,7 +47,7 @@ export const EditConnectionsContainer = ({
 
       await Promise.all(
         filteredNotes
-          .filter((note) => note.id.includes(TEMP_ID_PREFIX))
+          .filter((note) => !note.id)
```

Comment:
OK, I understand why they had `TEMP_ID_PREFIX` now. I think you should revert the changes here. You need to be able to create multiple connection notes before doing the final save. Here, you will overwrite old ones. I think it's OK to have `TEMP_ID_PREFIX + Date.now()` as it'll be changed by you in the core

Reply:
oh, my bad i forgot to check to create multiple connections note